superbench/cli/_handler.py

```diff
@@ -319,3 +321,65 @@ def run_command_handler(
     runner.run()
     if runner.get_failure_count() != 0:
         sys.exit(runner.get_failure_count())
+
+
+def info_command_handler(
+    docker_image='superbench/superbench',
+    docker_username=None,
+    docker_password=None,
+    no_image_pull=False,
+    host_file=None,
+    host_list=None,
+    host_username=None,
+    host_password=None,
+    output_dir=None,
+    private_key=None
+):
+    """Collect the system info on all given nodes.
+
+    Args:
+        docker_image (str, optional): Docker image URI. Defaults to superbench/superbench:latest.
+        docker_username (str, optional): Docker registry username if authentication is needed. Defaults to None.
+        docker_password (str, optional): Docker registry password if authentication is needed. Defaults to None.
+        no_image_pull (bool, optional): Skip pull and use local Docker image. Defaults to False.
+        host_file (str, optional): Path to Ansible inventory host file. Defaults to None.
+        host_list (str, optional): Comma separated host list. Defaults to None.
+        host_username (str, optional): Host username if needed. Defaults to None.
+        host_password (str, optional): Host password or key passphase if needed. Defaults to None.
+        output_dir (str, optional): Path to output directory. Defaults to None.
+        private_key (str, optional): Path to private key if needed. Defaults to None.
+
+    Raises:
+        CLIError: If input arguments are invalid.
+    """
+    # local
```

Comment:
do you still need "sb node info" for local case?

Reply:
need local command for ansible run in local machine and also user may just want to use local without ansible, according to previous discussion, merge them into one command is better for user?